From 55cae4e85d59b5c6cbfd00257833ccaab2b76982 Mon Sep 17 00:00:00 2001
From: Charles Short <charles.short@windriver.com>
Date: Wed, 13 Jul 2022 10:04:57 -0400
Subject: [PATCH] Disable systemd for dnsmasq

Disable systemd from starting at boot since its managed by the
sm service.

Signed-off-by: Charles Short <charles.short@windriver.com>
---
 debian/postinst | 16 ----------------
 1 file changed, 16 deletions(-)

diff --git a/debian/postinst b/debian/postinst
index 8c93160..38c2a07 100644
--- a/debian/postinst
+++ b/debian/postinst
@@ -4,22 +4,6 @@ set -e
 # /usr/share/doc/dnsmasq was a symlink in versions < 2.81-1 (see #985282)
 dpkg-maintscript-helper symlink_to_dir /usr/share/doc/dnsmasq dnsmasq-base 2.84-1.2~ dnsmasq -- "$@"
 
-# Code copied from dh_systemd_enable ----------------------
-# This will only remove masks created by d-s-h on package removal.
-deb-systemd-helper unmask dnsmasq.service >/dev/null || true
-
-# was-enabled defaults to true, so new installations run enable.
-if deb-systemd-helper --quiet was-enabled dnsmasq.service; then
-	# Enables the unit on first installation, creates new
-	# symlinks on upgrades if the unit file has changed.
-	deb-systemd-helper enable dnsmasq.service >/dev/null || true
-else
-	# Update the statefile to add new symlinks (if any), which need to be
-	# cleaned up on purge. Also remove old symlinks.
-	deb-systemd-helper update-state dnsmasq.service >/dev/null || true
-fi
-# End code copied from dh_systemd_enable ------------------
-
 if [ -x /etc/init.d/dnsmasq ]; then
    update-rc.d dnsmasq defaults 15 85 >/dev/null
 
-- 
2.25.1

